packages/db/src/sql/sqlite.ts

```
@@ -1041,307 +934,286 @@ class SQLite<T extends string> extends SQL<T> implements DbBackend<T> {
+            columnCount: columns.length,
+          });
 
-          resolve(columns)
-        }
-      )
-    })
+          resolve(columns);
+        },
+      );
+    });
   }
```

Biome (2.4.9)

[error] This async function lacks an await expression.

(lint/suspicious/useAwait)